### PR TITLE
[ticket/15592] posting.php changed to add a variable allowed_bbcode and posting_atta…

### PR DESCRIPTION
**Pull request by amalnaeem**
…ch_body.html changed to add a function to hide/display place inline button to resolve [ticket/15592]

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [ ] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15592

### CODE REVIEW

**Comment by hanakin** [comment]
1. you commits do not follow the proper commit format https://area51.phpbb.com/docs/dev/3.2.x/development/git.html 
2. pls do not inject js scripts into html body, this is a way outdated appraoch. This should be handled via binding and the data object, leverage jQuery

**Comment by amalnaeem** [comment]
Can you elaborate briefly the second point? It will be of much help. Thank you

**Comment by hanakin** [comment]
Yes use jQuery selectors and data objects instead of adding the code to the html? The js itself should be in core.js

File: phpBB/assets/javascript/core.js [hunk 1]
```diff
@@ -1676,3 +1676,19 @@ $(function() {
 });
 
 })(jQuery); // Avoid conflicts with other libraries
+$(function(){
+
+jQuery( document ).ready(function() {
```
Comment (hanakin) [inline phpBB/assets/javascript/core.js]: Do not need all the jQuerys just use $ and wrap it in a self invoked function with jQuery passed into like the example above this in the same file

Comment (hanakin) [reply phpBB/assets/javascript/core.js]: Also why are you double wrapping with document ready?

**Pull request closed without merging**